tests/test_core/test_sequence.py

```diff
@@ -2613,3 +2619,27 @@ def test_parent_coordinates(rev):
         seq = seq.rc()
 
     assert seq.parent_coordinates() == (0, 0, 1)
+
+
+def test_seqview_seqid():
+    sv = SeqView("ACGGTGGGAC")
+    assert sv._seqid is None
```

Comment:
change all references to the private attribute `_seqid` to the property `.seqid`